ui/imports/utils/Utils.qml

```diff
@@ -717,6 +697,38 @@ QtObject {
     /* Validation section end */
 
 
+    function getContactDetailsAsJson(publicKey) {
+        let jsonObj = mainModule.getContactDetailsAsJson(publicKey)
+        let obj = JSON.parse(jsonObj)
```

Comment:
Is there any chance the JSON might not be valid? If so, we can add a `try/catch`

Reply:
@jrainville json received here shouldn't be invalid ever, as we create it on the Nim side, but regardless to that I agree we should be correct in any case and I updated this and handled it in a try/catch. Done.